a compile error

With dce98d017d879d7c5a5128bd1b8b1322feb7acfa, we get a new script.
The file option_enum.cpp.in has a line:
```.cpp
#pragma GCC diagnostic ignored "-Wswitch-bool"
```
which is not correctly compiled at:
https://travis-ci.org/uncrustify/uncrustify/jobs/489179630, line 723:
```.txt
/home/travis/build/uncrustify/uncrustify/build/src/option_enum.cpp:22:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
 #pragma GCC diagnostic ignored "-Wswitch-bool"
                                ^
```
How can it be corrected?

--- a/src/option_enum.cpp.in
+++ b/src/option_enum.cpp.in
@@ -12,7 +12,7 @@
 
 #if defined(_MSC_VER)
 #pragma warning(disable: 4809)
-#elif defined(__GNUC__)
+#elif __GNUC__ > 4 || __clang_major__ > 3 || __clang_minor__ > 4
 #pragma GCC diagnostic ignored "-Wswitch-bool"
 #endif
 

